.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: python
+dist: trusty
+sudo: false
+cache: pip
+
+python:
+  - '3.6'
+  - '3.7-dev'
+  - 'nightly'
+
+matrix:
+  allow_failures:
+    - python: nightly
+  fast_finish: true
+
+install:
+  - pip install pylint
+  - pip install -r requirements.txt
+
+script:
+  # Display all lints and a report
+  - pylint --reports=yes futaba || true
```

Comment:
in general, it's tough to enforce anything if it doesn't fail the build. I think having this step vs not having it is the same in the long run, so you should instead decide what you want and fail the build when someone commits something you don't

Reply:
I think it's good to list all the warnings, but you're right, that's the only real enforcement. When the codebase is more mature I want to switch it to have any warnings fail the build.